extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -382,7 +386,8 @@ public static function onParserAfterTidy( Parser $parser, &$text ): bool {
 				[
 					'content' => $text,
 					'media' => self::$media,
-					'users' => self::$users
+					'users' => self::$users,
+					'heroImage' => empty( self::$heroImage ) ? null : self::$heroImage
```

Comment:
what do you think about setting heroImage as null by default
this way we would not have to have this ternary here

Reply:
👍